src/ConfigParser/ConfigParser.js

```diff
@@ -264,16 +264,12 @@ ConfigParser.prototype = {
          * @param  {string} density Density of resource.
          * @return {Resource}       Resource object or null if not found.
          */
-        ret.getByDensity = function (density) {
-            return ret.filter(function (res) {
-                return res.density === density;
-            })[0] || null;
+        ret.getByDensity = density => {
+            return ret.filter(res => res.density === density)[0] || null;
```

Comment:
Suggested refactoring idea, similar to comments above:

```js
        ret.getByDensity = density => (
            ret.filter(res => (res.density === density))[0] || null;
        );
```

and should we check the length before attempting to access the first element?

Reply:
The `|| null` already handles the case where `length === 0`.

Reply:
the `return` statement of `return ret.filter(res => res.density === density)[0] || null;` could in theory be omitted, too, as Brody said though.